public/views/partials/lib/validations/hcaptcha.liquid

```diff
@@ -0,0 +1,15 @@
+{% comment %}
+  params: @params_with_hcaptcha
+          @c
+{% endcomment %}
+
+{% liquid
+  if context.constants.VERIFY_HCAPTCHA == "true" and hcaptcha_params
```

Comment:
`if context.constants.VERIFY_HCAPTCHA == "true"` should be removed. looks like a poorly implemented feature flag, doesn't make sense in this context.